OWNERS

```diff
@@ -4,6 +4,7 @@ approvers:
 - rhuss
 - maximilien
 - navidshaikh
+- zhanggbj
```

Comment:
I wonder whether we could add a OWNERS fils within the `plugins/admin` directory ? I think this would be nice as it highlights the responsibilities and point-of-contact better. 

I think prow supports this, let me check ...

Reply:
Sounds good to me. Thanks a lot and I've joined Knative org.